local-cli/server/server.js

```diff
@@ -87,6 +91,10 @@ function _server(argv, config, resolve, reject) {
     )
     : config.getAssetRoots();
 
+  args.assetExts = args.assetExts
```

Comment:
I followed the convention from `projectRoots` and `assetRoots` above it -- I think if you specify a parameter here you want to override the config in `rn-cli.config.js`. If this seems wrong we should probably address in a separate PR that changes the behaviour for all of the params